gw_spaceheat/actors/synth_generator.py

```diff
@@ -59,6 +60,13 @@ def __init__(self, name: str, services: ServicesInterface):
         self.previous_time = None
         self.temperatures_available = False
 
+        self.pico_tanks = [node for node in self.layout.nodes.values() if node.ActorClass==ActorClass.ApiTankModule]
+        self.ab_by_pico = {}
+        for node in self.pico_tanks:
+            self.ab_by_pico[node.component.gt.PicoAHwUid] = 'a'
+            self.ab_by_pico[node.component.gt.PicoBHwUid] = 'b'
+        print(self.ab_by_pico)
+
```

Comment:
synth generator should not need to know how we are generating data channels - for example if we are measuring tank temperatures with some new temperature sensor we should not need to change synth gen